src/js/disability-benefits/526EZ/helpers.jsx

```diff
@@ -209,3 +209,52 @@ export const additionalDocumentDescription = () => {
   );
 };
 
+const documentLabels = {
+  1: 'Discharge',
+  2: 'Marriage related',
+  3: 'Dependent related',
+  // 4: 'VA preneed form',
+  5: 'Letter',
+  6: 'Other'
+};
+
+const getVACenterName = (center) => center.treatment.treatmentCenterName;
+const getPrivateCenterName = (release) => release.privateRecordRelease.treatmentCenterName;
+
+const listifyCenters = (center, idx, list) => {
+  const getName = center.treatment ? getVACenterName : getPrivateCenterName;
+  const notLast = idx < (list.length - 1);
+  const justOne = list.length === 1;
+  const atLeastThree = list.length > 2;
+  return (
+    <strong key={idx}>
```

Comment:
It's probably fine, but this will generate kind of odd markup:
```html
<li>We will get your medical records from <strong>Center One, </strong><strong>Center two, </strong><strong> and Center Three</strong>.</li>
```
Is there any way to make that
```html
<li>We will get your medical records from <strong>Center One, Center two, and Center Three</strong>.</li>
```
?

Reply:
What are your thoughts on the new markup (in light of @peggygannon's feedback)? (It's probably equivalently odd.)